students/dave_arasim/session04/mailroom2.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+'''Mailroom Part 2 for Session 4, Python 210
+Written by David K. Arasim - 10/28/18'''
+
+#Imported modules
+import platform
+import os
+from collections import defaultdict
+
+#Global variables
+this_os = platform.system()
+quit_option = False
+
+#############################################################################################
+#Function section
+
+def main():
+    global this_os
+    global quit_option
+
+    donor_dict = defaultdict(list)
+    build_donor_dict(donor_dict)    #Build the Donor Dictionary from a text file
+
+    switch_dict = {1: send_thank_you, 2: create_report, 3: send_all_thanks, 4: quit_option_true}
+
+    while not(quit_option):
```

Comment:
Minor: `not quit_option`, instead of `not(quit_option)`. `not` is not a function, but a unary (taking only one operand) operator.